products/health-care/appointments/va-online-scheduling/express-care-research/express-care-conversation-guide.md

```diff
@@ -0,0 +1,120 @@
+    - What would convince you to try it?
+  - Would you consider using Telehealth appointments for a same-day appointment?
+- Have you heard anything about same-day/urgent care Telehealth appointments through VA?
+  - (Clarification question if needed: Do you think that VA offers any same-day urgent care appointments? Do you think that the VA offers any same-day urgent care Telehealth appointments?)
+- Have you ever tried to make a same-day appointment with a VA provider, either someone you see regularly or any available provider?
+  - If YES: 
+    - Can you tell me what happened?
+    - How did you make this appointment?
+    - Were you able to be seen?
+  - If NO:
+    - Have you ever considered doing this, but something prevented you? If so, what?
+
+## Section 2: Scheduling expectations
+
+- Do you consider "same-day" or urgent care the same thing as scheduling an appointment? If not, how is this different?
+- Can you tell me what types of urgent care you might seek through VA's same-day Telehealth appointments?
+- Who do you expect to speak with for an urgent same-day VA Telehealth appointment?
+
+## Section 3: Coronavirus appointments 
+
+Next, I'm going to ask you some questions related to coronavirus. These are general questions related to how you might seek medical help if you are experiencing symptoms. If you feel uncomfortable at any point we can move on to the next section or stop, just let me know. 
+- Can you tell me in your own words what COVID-19 means?
+- When you are speaking with family or friends how do you refer to the current pandemic?
+  - Do you differentiate between the disease versus the virus?
+- Have you been practicing physical distancing, physical isolation, or quarantined, by choice or otherwise?
+  - If so, what caused you to do so?
+  - If not, what would cause you to do so?
+- Where do you go to seek guidance on taking care of yourself and your family during this pandemic?
+  - What VA health care resources are you aware of related to coronavirus, if any?
+- Have you had any concern that you have experienced symptoms related to coronavirus?
+  - If YES: 
+    - What did you do to get care?
+  - If NO: 
+    - Imagine that you did, what would your first step towards seeking care be?
+  - Are you aware of any coronavirus screening resources through VA available to you?
+- Would you use an online screening tool if you were experiencing symptoms of coronavirus?
+- Do you think VA has appointments available to help you if you are experiencing symptoms or may be suffering from coronavirus?
+  - Do you think VA has any urgent Telehealth appointments available to yourself or Veterans?
+
+## Section 4: [Prototype]
+
+Next, we're going to take a look at a prototype. As I mentioned earlier, this is a demo tool that may not function exactly the way you expect. Some areas of the prototype will be clickable, and some will not. Since it's a prototype, none of your actions will affect your actual VA information or benefits.
+
```

Comment:
A few questions for the prototype/to discuss on Monday.

1.  Should we start with the VA home page?
- If Yes, then I assume we need to ask where they think they might arrange a telehealth appointment would be? That would lead to more conversation.
- If No, then where to start?

2. Should we start directly on the Scheduling page? 
- If Yes, then I have a couple options already (full width alert banner, alert box within the current Scheduling home page)
- I'm thinking we can skip the "Sign in" UI - assume the Veteran is already signed in? Or do we want to test this flow too. Maybe VAOS has it covered already. 

3. We could add the "express care" option to both the Scheduling home page and VA Appointments page (similarly, I have a full width alert banner and an alert box option)

4. Then we can show the "schedule a telehealth request" form page.

5. Stop here? Or ask what they expect would happen after filling out the form?